arangod/RestServer/FlushFeature.h

```diff
@@ -68,8 +112,10 @@ class FlushFeature final : public application_features::ApplicationFeature {
 
   basics::ReadWriteLock _callbacksLock;
   std::unordered_map<void*, FlushCallback> _callbacks;
+  std::unordered_set<std::shared_ptr<FlushSubscriptionBase>> _flushSubscriptions;
+  std::mutex _flushSubscriptionsMutex;
```

Comment:
please rename `_flushSubscriptionsMutex` in to `_flushSubscriptionsLock`

Reply:
it's only a mutex, not a scoped lock though